### PR TITLE
FMDB now depends on SalesforceSDKCore for logging

### PR DESCRIPTION
**Pull request by bhariharan**

### CODE REVIEW

**Comment by bhariharan** [comment]
@khawkins @wmathurin

File: fmdb.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -642,6 +765,10 @@
 				GCC_WARN_UNINITIALIZED_AUTOS = YES;
 				GCC_WARN_UNUSED_FUNCTION = YES;
 				GCC_WARN_UNUSED_VARIABLE = YES;
+				HEADER_SEARCH_PATHS = (
+					"$(PROJECT_DIR)/../../libs/SalesforceSDKCore/SalesforceSDKCore/Classes/**",
+					"$(inherited)",
+				);
```
Comment (wmathurin) [inline fmdb.xcodeproj/project.pbxproj]: Shouldn't it be $(BUILT_PRODUCTS_DIR)/Headers $(TARGET_BUILD_DIR)/Headers $(inherited)

Comment (bhariharan) [inline fmdb.xcodeproj/project.pbxproj]: Yep - I fixed that.

**Comment by wmathurin** [comment]
:+1:

**Pull request merged**